docs/architecture/devops-for-aspnet-developers/actions-vs-pipelines.md

```diff
@@ -0,0 +1,121 @@
+---
+title: DevOps with .NET and GitHub Actions - Comparing GitHub Actions with Azure Pipelines
+description: GitHub Actions and Azure Pipelines compared and contrasted for decision makers
+author: colindembovsky
+ms.date: 03/04/2021
+---
+
+# Comparing and contrasting GitHub Actions and Azure Pipelines
+
+GitHub Actions and Azure Pipelines have a common history - in fact, the Actions agent is a fork of the Pipelines agent. There are many similarities between GitHub Actions and Azure Pipelines and it is worth comparing and contrasting them.
+
+## Pipelines as code
+
+Before you compare GitHub Actions and Azure Pipelines, you should consider the benefits of _pipelines as code_. Pipelines as code:
+
+> [!div class="checklist"]
+
+> * Benefit from standard source control practices (such as code reviews via pull request and versioning).
+> * Can be audited for changes just like any other files in the repository.
+> * Don’t require accessing a separate system or UI to edit.
+> * Can fully codify the build, test, and deploy process for code.
+> * Can usually be templatized to empower teams to create standard processes across multiple repositories.
+
+> [!INFORMATION]
```

Comment:
This won't render as expected. I see it's used in another article [here](https://github.com/dotnet/docs/blob/main/docs/architecture/containerized-lifecycle/docker-devops-workflow/docker-application-outer-loop-devops-workflow.md). I opened https://github.com/dotnet/docs/pull/23444 to fix it.

Reply:
Thanks @Youssef1313 - I'll update it to `NOTE`